docs/tutorials/tut0.rst

```diff
@@ -29,11 +29,11 @@ If Docker is not installed, you can get it `here <https://www.docker.com/get-sta
 
 .. code-block:: bash
 
-  sudo docker run --name tutorial -p 5433:5432 \
```

Comment:
Ah, this won't help us. Once I changed this to 5433 many reports stopped coming, going back to 5432 will bring them back.

The definite fix for this would be the INI file we talked about before plus a note on the tutorial to use `psql SERVICEFILE..`.

Reply:
Related https://github.com/PostgREST/postgrest-docs/issues/304

So just to be clear, the problem with 5432 is that most users already a pg running and using the same port will cause a port conflict.

Reply:
https://github.com/PostgREST/postgrest/pull/3220 already will reduce the chance of using a wrong pg.

Reply:
Ok cool. Let's just merge as is.